optimize/convex/lp/simplex.go

```diff
@@ -70,10 +70,10 @@ const (
 //
 // The Convert function can be used to transform a general LP into standard form.
 //
-// The input matrix A must have full rank and may not contain any columns with
-// all zeros. Furthermore, len(c) must equal the number of columns of A, and len(b)
-// must equal the number of rows of A. Simplex will panic if these conditions are
-// not met.
+// The input matrix A must have at least as many columns as rows, must have
+// full row rank and may not contain any columns with all zeros. Furthermore,
```

Comment:
> ... must have full row rank ...

If we do panic on this, I think it should be an error instead.

Reply:
Agreed. Fixed comment to match the actual behavior.